tests/suites/test_suite_ecp.function

```diff
@@ -1520,7 +1520,7 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS */
+/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS:MBEDTLS_ECP_WITH_MPI_UINT */
```

Comment:
We may need to update that dependency on the ecp.py [file](https://github.com/Mbed-TLS/mbedtls/blob/development/scripts/mbedtls_dev/ecp.py#L37). ( Multiple occurences )

Reply:
Does it mean we should move the test cases into the auto-generated file?